### ISSUE TITLE
How to use together with split-by-name-webpack-plugin?

### ISSUE THREAD

**Issue by bguiz** [markdalgleish/static-site-generator-webpack-plugin#42]
This plugin allows the build output to be split into multiple bundles: [split-by-name-webpack-plugin](https://github.com/soundcloud/split-by-name-webpack-plugin)

... however, when used together with static-site-generator-webpack-plugin, I get the following error:

```
ERROR in ReferenceError: webpackJsonp is not defined
    at app:11:8
    at webpackUniversalModuleDefinition (app:3:20)
    at app:10:3
    at ContextifyScript.Script.runInNewContext (vm.js:18:15)
    at module.exports (/code/my-project/node_modules/static-site-generator-webpack-plugin/node_modules/eval/eval.js:62:12)
    at Compiler.<anonymous> (/code/my-project/node_modules/static-site-generator-webpack-plugin/index.js:30:20)

...
```

Googling around, it looks it is similar to 
[this error encountered with CommonChunksPlugin](https://github.com/webpack/webpack-dev-server/issues/78)
and the solution could be related to not all chunks being included when attempting to generate.

Digging into the source, in `index.js`:

```
      var source = asset.source();
      var render = evaluate(source, /* filename: */ self.renderSrc, /* scope: */ undefined, /* includeGlobals: */ true);
```

It looks like assuming just 1 source here could be related to that. 

Is there a way to specify that this plugin should use multiple sources or multiple chunks?

**Comment by bguiz** [comment]
Added a PR #43 to show a way to build with multiple `renderSrc` (specified as an array).

This *almost* solved the problem for me, however, still failing because the way webpack splits chunks, assume that the `window` global is available:

```
var parentJsonpFunction = window["webpackJsonp"];
```

This results in output that is OK in the browser, but during the build, it fails 'cos NodeJs doesn't have it... any ideas?

**Comment by markdalgleish** [comment]
Looks like we need to close #31 first, then we can look at getting your PR in.

**Comment by markdalgleish** [comment]
Fixed in `v2.1.0` because you can now pass in a scope object as the last argument. In your case, you can provide `{ window: {} }` and it should work.

**Comment by ericmuyser** [comment]
Did that work for you @bguiz?
It's not working here AFAICT
